src/GraphQL/EnumerableExtensions.cs

```diff
@@ -8,6 +8,7 @@ public static class EnumerableExtensions
         /// <summary>
         /// Performs the indicated action on each item.
         /// </summary>
+        /// <param name="items">The list of items to act on.</param>
```

Comment:
Not a list.

Reply:
Huh?  It's not a `List<T>` I know, but it's a list......  

Please suggest a change

Reply:
I'm OK with it.